.env.docker

```
@@ -549,3 +549,42 @@
+  "responseTypesSupported": ["code"],
+  "scopesSupported": ["mcp.read", "mcp.write", "mcp.admin", "openid", "profile", "email"],
+  "codeChallengeMethodsSupported": ["S256"]
+}]'
+
+# Alternative: Token introspection for opaque tokens
+# MCP_AUTH_INTROSPECTION_ENDPOINT=http://localhost:3000/oauth2/introspect
+# MCP_AUTH_INTROSPECTION_CLIENT_ID=gauzy-mcp-dev-client
+# MCP_AUTH_INTROSPECTION_CLIENT_SECRET=dev-client-secret
+
+# Cache settings for performance
+MCP_AUTH_TOKEN_CACHE_TTL=300     # 5 minutes
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)